### ISSUE TITLE
Option to hide the 'Switch to YTM' button

### ISSUE THREAD

**Issue by TheRealSebTombs** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3765]
Please add an option to hide the Switch to YTM button, or to at least to customise it so it can be made dark, thank you.

**Comment by jostrander** [comment]
How about following the custom theme? 
No Theme:
![image](https://user-images.githubusercontent.com/760989/71634410-d9d1c880-2be9-11ea-9566-f5ced87ca3e1.png)
Themed Dark:
![image](https://user-images.githubusercontent.com/760989/71634417-e6562100-2be9-11ea-8598-d4a913c31c29.png)
Themed: Light:
![image](https://user-images.githubusercontent.com/760989/71634422-efdf8900-2be9-11ea-8f57-11eafd2474a2.png)

**Comment by TheRealSebTombs** [comment]
I'm themed dark, so the button sticks out whichever colour I choose. I'll try very dark grey though. Pity beause I liked a colour accent.

**Comment by TheRealSebTombs** [comment]
> How about following the custom theme?
> No Theme:
> ![image](https://user-images.githubusercontent.com/760989/71634410-d9d1c880-2be9-11ea-9566-f5ced87ca3e1.png)
> Themed Dark:
> ![image](https://user-images.githubusercontent.com/760989/71634417-e6562100-2be9-11ea-8598-d4a913c31c29.png)
> Themed: Light:
> ![image](https://user-images.githubusercontent.com/760989/71634422-efdf8900-2be9-11ea-8f57-11eafd2474a2.png)

Hmm. I am using a version that was compiled and shared to solve the 'stuck on loading screen issue' which is marked as Version 4.6.1 (4.6.1) but the theme doesn't affect the YT button, it stays bright red no matter which colour I choose. So choosing a dark grey wouldn't work.

But I am sure there are people out there who don't like YT Music, it would be nice to be able to hide the button if it's a service one doesn't use.

Thank you.

**Comment by jostrander** [comment]
These are changes I have made locally as I originally contributed the button to the project. I could go the route I did with YTM and make the background transparent and let the text follow the theme color instead, but YTM only has dark mode so that might not work in GPM.

**Comment by TheRealSebTombs** [comment]
> These are changes I have made locally as I originally contributed the button to the project. I could go the route I did with YTM and make the background transparent and let the text follow the theme color instead, but YTM only has dark mode so that might not work in GPM.

Thanks for your reply Jesse. I assume having the option to hide it altogether through a tick box in the Preferences would be too involved?

**Comment by Alipoodle** [comment]
Temporary Custom CSS methods of theming / removing.
```css
/* Removes new Switch to YT Music Button */
#ytm-button { display: none; }

/* Sets the colour to Accent Colour (Only good for Dark / Custom Theme */
#ytm-button { background: <<FORE_PRIMARY>>; }

/* Sets the Black with White Text */
#ytm-button { background: <<BACK_PRIMARY>> !important; color: <<FORE_PRIMARY>> !important; }

/* Sets to White with Black text */
#ytm-button { background: <<FORE_PRIMARY>> !important; color: <<BACK_PRIMARY>> !important; }

/* No Background */
#ytm-button { background: none !important; }
```
![image](https://i.imgur.com/SG8mHNv.png)
![image](https://i.imgur.com/CIfAzQF.png)
![image](https://i.imgur.com/POmiDJB.png)

I do think this is ultimately gonna be an issue with getting it to work on Most Themes Nicely, and I do feel that making the background Invisible is the cleanest and almost Laziest option to please most people.

**Comment by TheRealSebTombs** [comment]
Thank you! Will use this until further notice.

**Comment by Zomby2D** [comment]
Unfortunately, the "Switch to GPM" button cannot be hidden since custom styles for Youtube Music isn't available. So I'm stuck with that annoying button that I can't get rid of.
We most definitely need an option to remove it.

**Comment by caustin89** [comment]
@Alipoodle -> I'm here searching for a solution to hide this button, so great idea using this css as a temporary solution. 

@Zomby2D @MarshallOfSound 
Only yesterday I stumbled onto the midnight theme for Google Play Music, and I was thrilled. And now I've stumbled onto the desktop app, and I'm blown away. Well done. Seriously. 

Wondering, since the "Switch to GPM" button can't be hidden - why not consider alternative placement for it? Side menu, perhaps. Unless of course this toggle is something a lot of users are 
using.. then I will patiently wait for an update to hide permanently. The app is phenomenal as is.
